.github/workflows/ci.yml

```diff
@@ -289,6 +289,7 @@ jobs:
       - run: pytest -n 2 modin/pandas/test/dataframe/test_join_sort.py --execution=${{ matrix.execution }}
       - run: pytest -n 2 modin/pandas/test/dataframe/test_map_metadata.py --execution=${{ matrix.execution }}
       - run: pytest -n 2 modin/pandas/test/dataframe/test_reduction.py --execution=${{ matrix.execution }}
+      - run: pytest -n 2 modin/pandas/test/dataframe/test_resample.py --execution=${{ matrix.execution }}
```

Comment:
What's need to have the new test file `test_resample`? Why do you not use existing appropriate places?